docs/ref/contrib/admin/index.txt

```diff
@@ -1230,6 +1230,14 @@ subclass::
         default search method but uses an index. Currently this is only
         available for MySQL.
 
+    ``!``
+        Use the '!' operator to specify your own field lookup. For example
```

Comment:
Add a `.. versionadded:: 2.0` annotation and a mention in `docs/releases/2.0.txt`.  See our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).